tsdb/chunkenc/chunk.go

```diff
@@ -72,10 +73,21 @@ type Appender interface {
 }
 
 // Iterator is a simple iterator that can only get the next value.
+// Iterator iterates over the samples of a time series.
 type Iterator interface {
+	// Next advances the iterator by one.
+	Next() bool
+	// Seek advances the iterator forward to the first sample with the timestamp equal or greater than t.
+	// If current sample found by previous `Next` or `Seek` operation already has this property, Seek has no effect.
+	// Seek returns true, if such sample exists, false otherwise.
+	// Iterator is exhausted when the Seek returns false.
+	Seek(t int64) bool
+	// At returns the current timestamp/value pair.
+	// Before the iterator has advanced At behaviour is unspecified.
 	At() (int64, float64)
+	// Err returns the current error. It should be used only after iterator is
```

Comment:
Grammer is off here, it makes it seem like calling this will make Next/Seek return false.

s/so/that is/

Reply:
:+1: Thanks.